config/examples/Creality/Ender-3 Pro/BigTreeTech SKR Mini E3 2.0/Configuration_adv.h

```diff
@@ -1034,7 +1034,7 @@
 #if ENABLED(ASSISTED_TRAMMING)
 
   // Define positions for probe points.
-  #define TRAMMING_POINT_XY { {  20, 20 }, { 180,  20 }, { 180, 180 }, { 20, 180 } }
+  #define TRAMMING_POINT_XY { {  50, 50 }, { 185,  50 }, { 185, 185 }, { 50, 185 } }
```

Comment:
Since tramming points will depend on your probe type & placement relative to the nozzle, it's probably best to leave these as Marlin's defaults.

Reply:
We do want to tram where the screws are, if possible, so the default should probably go for those points, and then it can be adjusted for a probe that can't reach.

Reply:
See my comment in https://github.com/MarlinFirmware/Configurations/pull/872#issuecomment-1368146126

Stock screw locations on Ender-3/5 beds are `TRAMMING_POINT_XY { {  32.5, 32.5 }, { 202.5,  32.5 }, { 202.5, 202.5 }, { 32.5, 202.5 } }`, but they may not be reachable depending on where the probe is located relative to the nozzle.

Ex: If your probe is to the left of the hotend, you may not be able to reach the screw heads on the right side of the bed or if your probe is to the right of the hotend, you may not be able to reach the screw heads on the left side of the bed.

It really depends on your probe/mount/probe offsets.